package.json

```diff
@@ -47,11 +47,11 @@
     "jasmine-console-reporter": "^3.0.2",
     "prettier": "^1.16.0",
     "shx": "^0.3.2",
+    "ts-node": "^7.0.0",
     "tslint": "^5.12.1",
     "tslint-config-prettier": "^1.13.0",
     "tslint-immutable": "^5.1.0",
-    "ts-node": "^7.0.0",
     "typedoc": "^0.14.2",
-    "typescript": "~3.2"
+    "typescript": "^3.5.3"
```

Comment:
Please use ~ notation (fixed minor) since typescript does not respect the semver spec and includes bbreaking changes in minor releases